slackbot/manager.py

```diff
@@ -5,15 +5,14 @@
 from glob import glob
 from six import PY2
 from importlib import import_module
-from slackbot import settings
 from slackbot.utils import to_utf8
 
 logger = logging.getLogger(__name__)
 
 
 class PluginsManager(object):
-    def __init__(self):
-        pass
+    def __init__(self, settings):
+        self.settings = settings
```

Comment:
what's the rationale for passing in settings here vs the way it was being done before with ```from slackbot import settings```?  (or is this an artifact of the earlier now-removed refactor where we had different bot settings)